driver/razermousemat_driver.c

```diff
@@ -93,11 +95,23 @@ static ssize_t razer_attr_read_set_brightness(struct device *dev, struct device_
     struct usb_interface *intf = to_usb_interface(dev->parent);
     struct usb_device *usb_dev = interface_to_usbdev(intf);
     struct razer_report response;
-    struct razer_report report = razer_chroma_standard_get_led_brightness(VARSTORE, BACKLIGHT_LED);;
+    struct razer_report report = {0};
```

Comment:
Why not just `struct razer_report report;`? I see that the report doesn't get used in the new codepath. If you think this is better like this, please also update the rest of the occurences of `struct razer_report report;` to `= {0};` in a separate commit

Reply:
With the command `git grep "struct razer_report" | grep -v '=' | grep ';'` all instances should be found I think

Reply:
Using = {0}; is a zero initialization of the struct. Should be easy enough to update separately.

Reply:
Added a follow-up patch to find and replace.